CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing
+Before contributing code to this project, please check to see if a similar or identical GitHub issue is already open.
```

Comment:
Maybe this should be "Before opening a new issue, ..."

Reply:
Or maybe just 
```suggestion
Before contributing to this project, please check to see if a similar or identical GitHub issue is already open.
```
I think we want issues for all contribution types. At least that's what I want to be made clear here.

Reply:
I've merged this change in, but I'll leave this as unresolved for additional comments.